api-reference/10 UI Components/dxPagination/1 Configuration/allowedPageSizes.md

```diff
@@ -0,0 +1,11 @@
+---
+id: dxPagination.Options.allowedPageSizes
+type: Array<Number, String>
+default: [5, 10]
+---
+---
+##### shortDescription
+Specifies the available page sizes in the page size selector.
```

Comment:
With plural, we usually try to go without articles, unless we refer to a specific list (we know what items are in that list, how many items, etc.)

```suggestion
Specifies page sizes available in the page size selector.
```